root/layout/components/sidebar/CollectionLinks.js

```diff
@@ -20,33 +20,37 @@ type Props = {|
 |};
 
 const CollectionLinks = ({$c, entity}: Props) => {
-  const allCollections = $c.stash.all_collections;
-  if (!$c.user_exists || !allCollections) {
+  const numberOfCollections = $c.stash.number_of_collections || 0;
+  if (!$c.user_exists) {
     return null;
   }
   return (
-    <>
-      <h2 className="collections">
-        {l('Collections')}
-      </h2>
-      <CollectionList
-        addText={l('Add to a new collection')}
-        entity={entity}
-        noneText={l('You have no collections!')}
-        usersLink={
-          <EntityLink
-            content={texp.ln(
-              'Found in {num} user collection',
-              'Found in {num} user collections',
-              allCollections.length,
-              {num: allCollections.length},
-            )}
-            entity={entity}
-            subPath="collections"
-          />
-        }
-      />
-    </>
+    <CollectionList
+      addCollectionText={l('Add to a new collection')}
+      collaborativeCollections={$c.stash.collaborative_collections}
+      collaborativeCollectionsHeader={l('Collaborative collections')}
+      collaborativeCollectionsNoneText={
+        l('Not collaborating on any collections!')
```

Comment:
Should we be showing a "Collaborative collections" section in the sidebar if there are none? Most users won't be participating in any and telling them that on every page doesn't seem super useful, since it's not like they can take any action about it (you have to be added to one by someone else). (But the only downside is it taking up space and adding more work for translators.)

Reply:
I thought it might be nice as a way of showing that they are a thing, but you might have a point that the section would soon become more annoying than useful. Will look into it :)

Reply:
Hidden. I also added a "Global usage" heading to the "in X user collections" the bottom, because otherwise it didn't look very good. Not completely sure about the wording though!

Reply:
Hmm, might be clearer if it mirrors the wording of the other heading ("My collections"), so maybe "Other user collections" or similar if it's not too long.

Reply:
Changed it to "Other collections" (since it's under "My collections" and "Collaborative collections", it should be clear it's other users')